pkgs/by-name/kr/krename/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  fetchurl,
+  fetchFromGitLab,
+  lib,
+  extra-cmake-modules,
+  kdePackages,
+  taglib,
+  exiv2,
+  podofo,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krename";
+  version = "5.0.60-unstable-2025-09-02";
+
+  # For when the next stable release is made
+  # src = fetchurl {
+  #   url = "mirror://kde/stable/krename/${version}/src/krename-${finalAttrs.version}.tar.xz";
+  #   hash = "sha256-sjxgp93Z9ttN1/VaxV/MqKVY+miq+PpcuJ4er2kvI+0=";
+  # };
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "utilities";
+    repo = "krename";
+
+    rev = "5ad5f5a1f0a1c7573fa1b872a1472dec96fe0dd7";
+    hash = "sha256-fGNiIGGq10F71wh37aKDB7Q3fCxSXqttg/176LH3nVM=";
+  };
+
+  buildInputs = with kdePackages; [
+    exiv2
+    podofo
+    kio
+    kxmlgui
+    qtbase
+    qtdeclarative
+    qt5compat
+    taglib
+  ];
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    kdePackages.kdoctools
+    kdePackages.wrapQtAppsHook
+  ];
+
+  env.NIX_LDFLAGS = "-ltag";
+
+  meta = {
+    description = "Powerful batch renamer for KDE";
+    mainProgram = "krename";
+    homepage = "https://kde.org/applications/utilities/krename/";
+    license = lib.licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
Good point. I had a look at a few of the source files - they all mention gpl2 or later. *But* there are a few *other* license files here https://invent.kde.org/utilities/krename/-/tree/master/LICENSES?ref_type=heads - including gpl2 only, but without going through all the sources (some of which have *no* mention of any license) it's rather hard to tell. So based on the ones I saw, let's at least change this to `gpl2Plus`.

Reply:
Did a `rg SPDX-License-Identifier | awk '{ print $3 }' | sort -u` and all those have been added.